Dockerfile.alpine

```diff
@@ -62,8 +65,8 @@ RUN wget https://ftp.postgresql.org/pub/source/v${PG96}/postgresql-${PG96}.tar.b
 
 RUN cd postgresql-9.6.* && \
   ./configure --prefix=/usr/local-pg9.6 --with-openssl=no --without-readline --with-system-tzdata=/usr/share/zoneinfo --enable-debug=no CFLAGS="-Os" && \
-  make -j $(nproc) && \
-  make install-world && \
+  make -j $(nproc) world-bin && \
```

Comment:
Just read up on this: https://www.postgresql.org/message-id/E1lz2A5-0003oD-05%40gemulon.postgresql.org

Yeah, this looks like a good move.  No real need for including the docs for older releases in the image. :smile: